.travis.yml

```diff
@@ -0,0 +1,35 @@
+addons:
+  apt:
+    packages:
+      - nodejs
+language: python
+sudo: false
+python:
+  - 3.5
+env:
+  - TOX_ENV=django111
+  - TOX_ENV=django20
+  - TOX_ENV=django21
+  - TOX_ENV=django22
+matrix:
+  include:
+    - python: 3.5
+      env: TOX_ENV=csslint
```

Comment:
These could just go into the list in `env:` above, couldn't they?

Reply:
Done. Note: I had shamelessly cargo-culted this from `edx-ora2`